applications/ParticleMechanicsApplication/python_scripts/particle_solver.py

```diff
@@ -188,9 +189,9 @@ def Check(self):
 
 #
 #
-def CreateSolver(model_part1, model_part2, new_element, config, geometry_element, num_par):
+def CreateSolver(model_part1, model_part2, model_part3, new_element, config, geometry_element, number_particle):
```

Comment:
my advice would be to give a more meaningful name to the different model parts.

not a blocker...but it would save your time later on